functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
```

Comment:
```suggestion
    for i in range(len(n)):
```
 for k in range(len(n)):

Reply:
I IS BETTER THAN K NO THANK U

Reply:
ok I it is